.github/workflows/haskell.yml

```diff
@@ -67,50 +67,48 @@ jobs:
       fail-fast: false
       matrix:
         os:  [ 'ubuntu-latest', 'macOS-latest', 'windows-latest' ]
-        ghc: [ '8.10', '9.0', '9.2', '9.4', '9.6' ]
+        ghc: [ '8.4', '9.4', '9.6', '9.8' ]
```

Comment:
Should '8.4' be '8.10'?

Reply:
Well, the proposal here is to test
- the latest 3 major GHCs, and
- the oldest GHC that is supported cross all OSs.

The latter turned out to be GHC 8.4.